data/Makefile.am

```diff
@@ -33,8 +36,16 @@ desktop_DATA =						\
 	setup-shell.desktop				\
 	$(NULL)
 
+demovideodir = $(datadir)/eos-demo-mode
+demovideo_DATA = \
+	eos-demo-video.ogv \
+	eos-demo-video.desktop \
```

Comment:
Hmm, I guess that's fair enough, though I guess there's the risk that we mess up the autostart check. I'll look into doing it that way...